python/llm/src/ipex_llm/transformers/npu_models/qwen2_mp.py

```diff
@@ -234,7 +234,7 @@ def __init__(
         print(f"{mode} start compiling")
         if (
             group_size != 0
-            and (mode == "prefill" or num_layers == 2)
+            and (mode == "prefill" or num_layers == 2 or num_layers == 3)
```

Comment:
We should unify the compilation configs with that in `update_names_of_IR_and_export_blob`

Reply:
We can do that in a future PR.

Reply:
Got it, will try to figure out how to do it and support it in next PR.